src/template.rs

```diff
@@ -54,11 +54,11 @@ impl FunctionTemplate {
                   callback: F) -> FunctionTemplate
         where F: Fn(value::FunctionCallbackInfo) -> value::Value {
         let raw = unsafe {
-            let callback = Box::into_raw(Box::new(callback));
+            let callback_ptr: *mut Box<F> = Box::into_raw(Box::new(Box::new(callback)));
```

Comment:
Sorry for missing this! I wonder how the tests passed? I will do reviews from now on haha